fairroot/online/steer/FairRunOnline.cxx

```diff
@@ -185,7 +185,6 @@ void FairRunOnline::Init()
     fTask->SetParTask();
     fRtdb->initContainers(fRunId);
 
-    //  InitContainers();
     // --- Get event header from Run
     if (!fEvtHeader) {
         LOG(fatal) << "FairRunOnline::InitContainers:No event header in run!";
```

Comment:
Can you please fix those LOG messages? They reference InitContainers.
(There's another one a few lines down.)

Reply:
You are right, I have changed it and added ChangeLog notice.